Nos/Views/NoteButton.swift

```diff
@@ -13,20 +13,46 @@ struct NoteButton: View {
     var style = CardStyle.compact
     var shouldTruncate: Bool
     var hideOutOfNetwork: Bool
-    var showReplyCount: Bool
-    var displayRootMessage: Bool 
-    var isTapEnabled: Bool 
+    var repliesDisplayType: RepliesDisplayType
+
+    /// Whether replies should be fetched from relays.
+    var fetchReplies: Bool
+
+    var displayRootMessage: Bool
+    var isTapEnabled: Bool
+    
     private let replyAction: ((Event) -> Void)?
     private let tapAction: ((Event) -> Void)?
+    @State private var relaySubscriptions = SubscriptionCancellables()
 
+    @EnvironmentObject private var relayService: RelayService
     @EnvironmentObject private var router: Router
     
+    /// Initializes a NoteButton object.
+    ///
+    /// - Parameter note: Note event to display.
+    /// - Parameter style: Card style. Defaults to `comapact`.
```

Comment:
```suggestion
    /// - Parameter style: Card style. Defaults to `.compact`.
```

Reply:
I also updated `displayNothing` to `.displayNothing` below to match your style of starting with the dot